docker/production/x86_64/Dockerfile

```diff
@@ -1,21 +1,25 @@
 FROM ubuntu:18.04 as prep
 LABEL MAINTAINER="leopere [at] nixc [dot] us"
 
+ARG STASH_BINARY=stash-linux
+ARG FFMPEG_ARCH=amd64
+ARG BASE_IMAGE=ubuntu
+
 RUN apt-get update && \
-    apt-get -y install curl xz-utils && \
+    apt-get -y install curl xz-utils ca-certificates && \
     apt-get autoclean -y && \
     rm -rf /var/lib/apt/lists/*
 WORKDIR /
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN curl -L -o /stash $(curl -s https://api.github.com/repos/stashapp/stash/releases/latest | awk '/browser_download_url/ && /stash-linux/' | sed -e 's/.*: "\(.*\)"/\1/') && \
+RUN curl -L -o /stash $(curl -s https://api.github.com/repos/stashapp/stash/releases/latest | awk '/browser_download_url/ && /$STASH_BINARY/' | sed -e 's/.*: "\(.*\)"/\1/') && \
     chmod +x /stash && \
-    curl --http1.1 -o /ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz && \
+    curl --http1.1 -o /ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-${FFMPEG_ARCH}-static.tar.xz && \
     tar xf /ffmpeg.tar.xz && \
     rm ffmpeg.tar.xz && \
     mv /ffmpeg*/ /ffmpeg/
 
-FROM ubuntu:18.04 as app
-RUN apt-get update && apt-get -y install ca-certificates
```

Comment:
any reason to change that?
copying the certs as you do later should work but isn't it better to have the package installed?

Reply:
Fair enough. Seems ok then.